### ISSUE TITLE
Run FNPRMS on fennecNightly builds on G5

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/perf-frontend-issues#69]
Our results currently come from Fenix Nightly but fennec* is what users will be receiving.

Note that we'll need to modify the launcher intent to use MigrationDecisionActivity or /.App instead of hard-coding HomeActivity (and perhaps similar actions for app link).

**Comment by mcomella** [comment]
The builds are available at https://firefox-ci-tc.services.mozilla.com/tasks/index/project.mobile.fenix.v2.fennec-nightly (I believe)

**Comment by bdekoz** [comment]
Looks like the testing machine in SF is up and giving reliable numbers for P2, P4xl, s10 for both the old binary (fenix-nightly) and fennec-nightly, as requested. The results repo is here: 

https://github.com/bdekoz/FNPRMS-results

If this looks ok, then let's close this.

**Comment by mcomella** [comment]
@ecsmyth Would you be able to validate that the results look okay? I think you have the most context on these numbers as you integrate them into your dashboard.

**Comment by ecsmyth** [comment]
I recommend that we keep this issue open until the `pageStop` revision is merged into @bdekoz's repo and we have new numbers

**Comment by mcomella** [comment]
I opened 2 PRs for this and I'm waiting for the last to land.

**Comment by mcomella** [comment]
One more PR to correct oversights from the previous PR: https://github.com/mozilla-mobile/FNPRMS/pull/20

**Comment by mcomella** [comment]
MarcLeclair re-ran the numbers and it looks reasonable. If we trust these values (I'll do a quick skim), we should ask bdekoz to make similar changes.

**Comment by mcomella** [comment]
I've verified the validity of our FNPRMS numbers through manual testing:
- Ensuring we're sending the correct intents
- Ensuring we're parsing the logs correctly
- Manually calculating the average fully drawn time (in python) for the 2020.03.15-hanoob.log and verifying it matched CI
- Generally skimming over other scripts to ensure nothing else interferes with our numbers

I've identified two bugs that do not impact our G5 Fenix numbers:
- Fennec 68 baseline may be lower than actual time #92 
- HANOOB Fenix parsing is hard-coded to work on G5-like devices #91

The second issue issue impacts our ability to run on non-G5 (i.e. bdekoz) and running locally. Since we've finished the most important thing in this issue, I'm going to close this issue and file a follow-up for the bdekoz machine: #93.
